docs/docs/migration.md

```diff
@@ -1,35 +1,35 @@
 
 
 #### Features that handled differently in the Cloud MTA Build Tool
 
-* Packaging of html5 modules in `deploy_mode=html5-repo`
-You need to update your `mta.yaml` file to exclude `html5` modules from the result MTA archive. For that, add the following to the `build-parameters` section of each  module of this type:
+* Packaging of HTML5 modules in `deploy_mode=html5-repo`
+You need to update your `mta.yaml` file to exclude `html5` modules from the resulting MTA archive. In order to do that, add the following to the `build-parameters` section for each  module of this type:
 
 ```yaml
 
    build-parameters:
       supported-platforms: []
 ```
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; For more details on the `supported-platforms` build parameter, see [Configuring and Packaging Modules According to Target Platforms](configuration.md#configuring-and-packaging-modules-according-to-target-platforms)
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; For more information about the `supported-platforms` build parameter, see [Configuring and Packaging Modules According to Target Platforms](configuration.md#configuring-and-packaging-modules-according-to-target-platforms)
   
-* The following `build-rarameters` are not supported by the Cloud MTA Build Tool: <ul><li>`npm-opts`<li>`grunt-opt`<li>`maven-opts`</ul>
+* The following `build-parameters` are not supported by the Cloud MTA Build Tool: <ul><li>`npm-opts`<li>`grunt-opt`<li>`maven-opts`</ul>
 
-  If you need to change default build behaviour defined for the correponding builder, please use [configure `custom` builder](configuration.md#configuring-the-custom-builder)
-  For complete list of available builders and their default behaviour, see [Builders execution commands](https://github.com/SAP/cloud-mta-build-tool/blob/master/configs/builder_type_cfg.yaml).
+  If you need to change the default build behavior defined for the correponding builder, see [configure `custom` builder](configuration.md#configuring-the-custom-builder).
+  For a complete list of available builders and their default behaviors, see [Builders execution commands](https://github.com/SAP/cloud-mta-build-tool/blob/master/configs/builder_type_cfg.yaml).
   <br>
 
 #### New features in the Cloud MTA Build Tool
 
-* If you would like to run some builder process before running builders of the specific modules, define it in [global `before-all` build parameters](configuration.md#configuring_before_all_build)
-* You can define your own build commands by [configuring `custom` builder](configuration.md#configuring-the-custom-builder)
+* If you want to run a builder process before running builders of the specific modules, define it according to [global `before-all` build parameters](configuration.md#configuring_before_all_build).
```

Comment:
"according to" does not fit here. Probably "using" is the right word. I will change it.

Reply:
How about: "...define it by using..."